boot/zephyr/boards/nrf54l15pdk_nrf54l15_cpuapp.conf

```diff
@@ -7,6 +7,4 @@ CONFIG_BOOT_MAX_IMG_SECTORS=256
 # Ensure that the qspi driver is disabled by default
 CONFIG_NORDIC_QSPI_NOR=n
 
-# TODO: below are not yet supported and need fixing
-CONFIG_FPROTECT=n
```

Comment:
Fprotect is not supported yet in nrf54l, so to avoid errors:
`CMake Error at /jenkins_workspace/workspace/latest_sdk-nrf_PR-14626/nrf/lib/fprotect/CMakeLists.txt:15 (message):`
`No fprotect backend selected.`
just keep that config (as [nrf noup] in sdk-mcuboot)

Reply:
can you please submit a patch?

Reply:
it was added in Upstream, but CONFIG_FPROTECT is defined in sdk-nrf